src/main/java/service/impl/FruitShopServiceImpl.java

```diff
@@ -0,0 +1,42 @@
+package service.impl;
+
+import dao.FruitDao;
+import dao.FruitDaoImpl;
+import java.util.List;
+import java.util.Map;
+import model.FruitOperation;
+import model.FruitType;
+import service.CalculateFruit;
+import service.FruitOperationParser;
+import service.FruitOperationTypeParser;
+import service.FruitShopService;
+import service.FruitTypeParser;
+import service.ReadDataFromFileSystem;
+import service.SaveToFileSystem;
+import service.SaveToStorage;
+
+public class FruitShopServiceImpl implements FruitShopService {
+    private final ReadDataFromFileSystem readDataFromFileSystem = new ReadDataFromFileSystemImpl();
+    private final CalculateFruit calculateFruitImpl = new CalculateFruitImpl();
+
+    private final FruitOperationTypeParser fruitOperationTypeParser
+            = new FruitOperationTypeParserImpl();
+    private final FruitTypeParser fruitTypeParser = new FruitTypeParserImpl();
+    private final FruitOperationParser fruitOperationParser
+            = new FruitOperationParserImpl(fruitOperationTypeParser, fruitTypeParser);
+
+    private final FruitDao fruitDao = new FruitDaoImpl();
+    private final SaveToStorage saveToStorage = new SaveToStorageImpl(fruitDao);
+    private final SaveToFileSystem saveToFileSystem = new SaveToFileSystemImpl(fruitDao);
+
+    @Override
+    public void processData(String readFromFileName, String writeToFileName) {
+        List<String> listOfData = readDataFromFileSystem.getData(readFromFileName);
+        List<FruitOperation> listFruitOperation
+                = fruitOperationParser.parseFruitOperationList(listOfData);
+        Map<FruitType, Integer> listCountedFruit
+                = calculateFruitImpl.calculateFruit(listFruitOperation);
```

Comment:
this method should not return anything, it should write to DB directly
![image](https://github.com/mate-academy/jv-fruit-shop/assets/73041091/fdd5e36f-fb82-4caf-b0a3-48c1bee07cd9)

```suggestion
        fruitCalculator.calculateFruits(fruits);
```